src/docfx/config/ops/OpsConfigLoader.cs

```diff
@@ -132,7 +132,12 @@ private static (JObject obj, string path, string name)[] GetDependencies(OpsConf
                 foreach (var ecma2Yaml in ecma2YamlConfig)
                 {
                     var ecma2YamlJObject = JsonUtility.ToJObject(ecma2Yaml);
-                    ecma2YamlJObject[nameof(ECMA2YamlRepoConfig.SourceXmlFolder)] = Path.GetRelativePath(buildSourceFolder, ecma2Yaml.SourceXmlFolder);
+                    var sourceXmlFolder = Path.GetRelativePath(buildSourceFolder, ecma2Yaml.SourceXmlFolder);
+                    if (!Directory.Exists(Path.GetFullPath(Path.Combine(docsetPath, sourceXmlFolder))))
```

Comment:
sourceXmlFolder is not always relative to docsetPath, it could be relative to repo root.

Reply:
As offline discussed, we get the relative path to `buildSourceFolder` firstly, which is `docsetPath`.